dev/tools/controllerbuilder/pkg/codegen/mappergenerator.go

```diff
@@ -174,8 +174,11 @@ func (v *MapperGenerator) GenerateMappers() error {
 			krmPackage := pair.KRMType.GoPackage
 
 			out.contents.WriteString(fmt.Sprintf("package %s\n\n", lastGoComponent(goPackage)))
-			out.contents.WriteString(fmt.Sprintf("import pb %q\n\n", pbPackage))
-			out.contents.WriteString(fmt.Sprintf("import krm %q\n\n", krmPackage))
+			out.contents.WriteString("import (\n")
+			out.contents.WriteString(fmt.Sprintf("\tpb %q\n", pbPackage))
+			out.contents.WriteString(fmt.Sprintf("\tkrm %q\n", krmPackage))
+			out.contents.WriteString(fmt.Sprintf("\t%q\n", "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/controller/direct"))
```

Comment:
Why is this a string we need to manually validate? Wouldn't it be better to use direct.GoPackage ?

Reply:
Not sure I follow - did this comment get moved between lines?